2022-03-28 11:34:54,755 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-28 11:34:54,755 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-28 11:34:54,755 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-28 11:34:54,755 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-28 11:34:54,755 INFO    MainThread:12299 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220328_113454-32srmuav/logs/debug.log
2022-03-28 11:34:54,755 INFO    MainThread:12299 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220328_113454-32srmuav/logs/debug-internal.log
2022-03-28 11:34:54,755 INFO    MainThread:12299 [wandb_init.py:init():439] calling init triggers
2022-03-28 11:34:54,755 INFO    MainThread:12299 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-28 11:34:54,755 INFO    MainThread:12299 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 7lwd3gen
2022-03-28 11:34:54,756 INFO    MainThread:12299 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/7lwd3gen
2022-03-28 11:34:54,756 INFO    MainThread:12299 [jupyter.py:save_history():429] not saving jupyter history
2022-03-28 11:34:54,756 INFO    MainThread:12299 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-28 11:34:54,756 INFO    MainThread:12299 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-28 11:34:54,757 INFO    MainThread:12299 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-28 11:34:54,758 INFO    MainThread:12299 [wandb_run.py:_restore():1674] restore
2022-03-28 11:34:55,085 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 11:34:55,216 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 11:34:55,405 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 11:34:56,168 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 143601
}

2022-03-28 11:34:56,277 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 143601
}

2022-03-28 11:34:56,381 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 143601
}

2022-03-28 11:34:56,492 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143601
  total_bytes: 143601
}

2022-03-28 11:34:56,606 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143601
  total_bytes: 143601
}

2022-03-28 11:34:56,728 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143601
  total_bytes: 143601
}

2022-03-28 11:34:56,840 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143601
  total_bytes: 143601
}

2022-03-28 11:34:56,948 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143601
  total_bytes: 143601
}

2022-03-28 11:34:57,057 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143601
  total_bytes: 143601
}

2022-03-28 11:34:58,020 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143601
  total_bytes: 143601
}

2022-03-28 11:34:58,485 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143601
  total_bytes: 143601
}
local_info {
}

2022-03-28 11:34:59,655 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-28 11:34:59,657 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-28 11:34:59,660 INFO    MainThread:12299 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-28 11:34:59,662 INFO    MainThread:12299 [wandb_init.py:init():492] starting backend
2022-03-28 11:34:59,662 INFO    MainThread:12299 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-03-28 11:34:59,664 INFO    MainThread:12299 [backend.py:ensure_launched():219] starting backend process...
2022-03-28 11:34:59,686 INFO    MainThread:12299 [backend.py:ensure_launched():224] started backend process with pid: 15212
2022-03-28 11:34:59,688 INFO    MainThread:12299 [wandb_init.py:init():501] backend started and connected
2022-03-28 11:34:59,710 INFO    MainThread:12299 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-28 11:34:59,710 INFO    MainThread:12299 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-28 11:34:59,710 INFO    MainThread:12299 [wandb_init.py:init():565] updated telemetry
2022-03-28 11:34:59,745 INFO    MainThread:12299 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-28 11:35:00,810 INFO    MainThread:12299 [wandb_run.py:_on_init():1759] communicating current version
2022-03-28 11:35:00,862 INFO    MainThread:12299 [wandb_run.py:_on_init():1763] got version response 
2022-03-28 11:35:00,862 INFO    MainThread:12299 [wandb_init.py:init():625] starting run threads in backend
2022-03-28 11:35:03,098 INFO    MainThread:12299 [wandb_run.py:_console_start():1733] atexit reg
2022-03-28 11:35:03,099 INFO    MainThread:12299 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-28 11:35:03,099 INFO    MainThread:12299 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-28 11:35:03,099 INFO    MainThread:12299 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-28 11:35:03,099 INFO    MainThread:12299 [wandb_init.py:init():664] run started, returning control to user process
2022-03-28 11:35:03,100 INFO    MainThread:12299 [wandb_watch.py:watch():47] Watching
2022-03-28 11:35:29,028 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set batch_size = 20 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83a1455b80>>
2022-03-28 11:35:29,028 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb batch_size 20 None
2022-03-28 11:35:29,028 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set epochs = 15 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83a1455b80>>
2022-03-28 11:35:29,029 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb epochs 15 None
2022-03-28 11:35:29,029 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set lr = 7.864941399192223e-05 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83a1455b80>>
2022-03-28 11:35:29,029 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb lr 7.864941399192223e-05 None
2022-03-28 11:35:29,029 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set no_cuda = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83a1455b80>>
2022-03-28 11:35:29,029 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb no_cuda False None
2022-03-28 11:35:29,029 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set seed = 42 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83a1455b80>>
2022-03-28 11:35:29,029 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb seed 42 None
2022-03-28 11:35:56,019 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-28 11:35:56,019 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-28 11:35:56,019 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-28 11:35:56,019 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-28 11:35:56,019 INFO    MainThread:12299 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220328_113556-f389yvyt/logs/debug.log
2022-03-28 11:35:56,019 INFO    MainThread:12299 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220328_113556-f389yvyt/logs/debug-internal.log
2022-03-28 11:35:56,019 INFO    MainThread:12299 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f83a1758e80>
2022-03-28 11:35:56,019 INFO    MainThread:12299 [wandb_init.py:init():439] calling init triggers
2022-03-28 11:35:56,019 INFO    MainThread:12299 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-28 11:35:56,020 INFO    MainThread:12299 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 32srmuav
2022-03-28 11:35:56,021 INFO    MainThread:12299 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/32srmuav
2022-03-28 11:35:56,021 INFO    MainThread:12299 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-28 11:35:56,022 INFO    MainThread:12299 [wandb_run.py:_restore():1674] restore
2022-03-28 11:35:56,367 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 11:35:56,608 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 11:35:57,088 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 143511
}

2022-03-28 11:35:57,198 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 143511
}

2022-03-28 11:35:57,307 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 143511
}

2022-03-28 11:35:57,411 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 143511
}

2022-03-28 11:35:57,519 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143511
  total_bytes: 143511
}

2022-03-28 11:35:57,632 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143511
  total_bytes: 143511
}

2022-03-28 11:35:57,741 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143511
  total_bytes: 143511
}

2022-03-28 11:35:57,869 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143511
  total_bytes: 143511
}

2022-03-28 11:35:57,979 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143511
  total_bytes: 143511
}

2022-03-28 11:35:58,094 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143511
  total_bytes: 143511
}

2022-03-28 11:35:58,204 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143511
  total_bytes: 143511
}

2022-03-28 11:35:58,861 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143511
  total_bytes: 143511
}

2022-03-28 11:35:59,464 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143511
  total_bytes: 143511
}
local_info {
}

2022-03-28 11:36:00,622 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-28 11:36:00,624 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-28 11:36:00,627 INFO    MainThread:12299 [wandb_run.py:_footer_sync_info():2822] logging synced files
